pkg/capr/planner/certificaterotation_test.go

```diff
@@ -118,7 +118,7 @@ func Test_rotateCertificatesPlan(t *testing.T) {
 					},
 					[]string{},
 				)}[0],
-				otiCount:   9, // the extra removal instructions are for removing the static pod manifests for RKE2
```

Comment:
I ran the tests on 2.8-head and encountered the following:
```
=== CONT  Test_rotateCertificatesPlan/test_KCM_cert_regeneration_removal_instruction_contains_RKE2
time="2023-09-12T19:14:56-04:00" level=error msg="rkecluster /: unable to find cluster spec annotation for control plane"
time="2023-09-12T19:14:56-04:00" level=error msg="Error while generating etcd snapshot extra metadata manifest for cluster "
time="2023-09-12T19:14:56-04:00" level=error msg="rkecluster /: unable to find cluster spec annotation for control plane"
time="2023-09-12T19:14:56-04:00" level=error msg="Error while generating etcd snapshot extra metadata manifest for cluster "
time="2023-09-12T19:14:56-04:00" level=error msg="rkecluster /: unable to find cluster spec annotation for control plane"
time="2023-09-12T19:14:56-04:00" level=error msg="Error while generating etcd snapshot extra metadata manifest for cluster "
time="2023-09-12T19:14:56-04:00" level=error msg="rkecluster /: unable to find cluster spec annotation for control plane"
time="2023-09-12T19:14:56-04:00" level=error msg="rkecluster /: unable to find cluster spec annotation for control plane"
time="2023-09-12T19:14:56-04:00" level=error msg="Error while generating etcd snapshot extra metadata manifest for cluster "
time="2023-09-12T19:14:56-04:00" level=error msg="Error while generating etcd snapshot extra metadata manifest for cluster "
time="2023-09-12T19:14:56-04:00" level=error msg="rkecluster /: unable to find cluster spec annotation for control plane"
time="2023-09-12T19:14:56-04:00" level=error msg="Error while generating etcd snapshot extra metadata manifest for cluster "
time="2023-09-12T19:14:56-04:00" level=error msg="rkecluster /: unable to find cluster spec annotation for control plane"
time="2023-09-12T19:14:56-04:00" level=error msg="Error while generating etcd snapshot extra metadata manifest for cluster "
```
Are these errors still present?

Reply:
These errors were in 2.8-head and are still present